modules/OFStateManager/module/src/ofstatemanager_int.h

```diff
@@ -80,12 +80,16 @@ calc_duration(indigo_time_t current_time, indigo_time_t entry_time,
 /**
  * @brief Opaque handle to an operation
  */
+#define INDIGO_ASYNC_OP_CTX_MAGIC     0xaabbccdd
 typedef struct indigo_core_op_context {
+    uint32_t magic;           /* memory protection. Internal usage. */
     indigo_cxn_id_t cxn_id;
     of_object_t *obj;
+    bool free_obj;            /* free the object in resume call */
     bool no_async;            /* indication this op context is not called for async operation.
                                * It is used in the OFStatemManager internally.
                                */
+    bool invalid_ctx;         /* OFStateManager internal usage */
```

Comment:
suggest removing "invalid_ctx" field and generally avoid introducing code that is unused -- this should help reduce future maintenance load

Reply:
This field is used to tell the ofstatemanager that op_ctx is not dynamic allocation in the clear table case. Maybe the variable name is not good. We can change it. I will remove the magic field.

Reply:
"invalid_ctx" doesn't seem be used anywhere in this pull request.  if you need it in another pull request, let's add it then, together with the code that actually uses it.